learning_observer/learning_observer/incoming_student_event.py

```diff
@@ -319,6 +319,7 @@ async def incoming_websocket_handler(request):
 
     INIT_PIPELINE = settings.settings.get("init_pipeline", True)
     json_msg = None
+    ws.send_str('Acknowledgment received')
```

Comment:
I'm not sure the purpose of this acknowledgement. The only reason I kind of know what it does is because I recall you describing like this during a meeting. I think it was to make sure both the client and server had seen a message, but please expand on why its needed in a comment or docstring.

Reply:
Yes, it was for testing and I have removed it